soh/soh/Enhancements/randomizer/randomizer.h

```diff
@@ -33,14 +39,15 @@ class Randomizer {
     void LoadHintLocations(const char* spoilerFileName);
     void LoadItemLocations(const char* spoilerFileName, bool silent);
     u8 GetRandoSettingValue(RandomizerSettingKey randoSettingKey);
-    RandomizerCheck GetCheckFromActor(s16 actorId, s16 actorParams, s16 sceneNum);
+    RandomizerCheck GetCheckFromActor(s16 sceneNum, s16 actorId, s16 actorParams);
     std::string GetChildAltarText() const;
     std::string GetAdultAltarText() const;
     std::string GetGanonText() const;
     std::string GetGanonHintText() const;
     std::string GetHintFromCheck(RandomizerCheck check);
     GetItemID GetRandomizedItemIdFromKnownCheck(RandomizerCheck randomizerCheck, GetItemID ogId);
     GetItemID GetRandomizedItemId(GetItemID ogId, s16 actorId, s16 actorParams, s16 sceneNum);
+    bool CheckContainsRandoItem(RandomizerCheck randoCheck);
```

Comment:
Yeah that's actually exactly why I didn't put the word Exclusive in it.

Reply:
The rando exclusive items have their modIndex in their ItemEntry so the game can differentiate that at Item_Give time for the rando exclusive items. This is what the code uses to determine which table to pull from, but still may pull a vanilla item from the rando table, which didn't have a vanilla getItemEntry because it was `Item_Give`en manually.

Reply:
hmm, what if it was `CheckContainsVanillaItem` and then the places where it's used to see if something should use the rando table or not we can use `!` (unless you foresee issues with that with future mods)

Reply:
Most likely randomizer is the only one that will use this, most other mods will probably just call ItemTable_RetrieveEntry or something like that since they won't have the same check return different items in different situations, and if they do they will probably have their own logic for determining that. Inverting that function to check for Vanilla Items may be better then, so we don't have to update that function when implementing any new `RG_` values down the line.

Reply:
> so we don't have to update that function when implementing any new RG_ values down the line.

hadn't thought about needing to update the function, but yeah, good call. inverting it seems like a good plan

Reply:
Addressed by 2077636c920ccd59c0ff864a7cbab28682e30467